packages/components/src/accordion/src/Accordion.css

```diff
@@ -15,7 +15,7 @@
 /* HEADER */
 .o-ui-accordion .o-ui-accordion-header {
     margin-bottom: 0;
-    font-variation-settings: "wght" 500;
+    font-weight: 505;
```

Comment:
505?

Reply:
With Inter 505 is "way" bolder than 500 oddly, I started with the Accordion component and went probably too far with my changes, as each components will be rebranded anyways and use the right token.